src/main/java/nl/tudelft/dnainator/ui/drawables/phylogeny/CollapsedNode.java

```diff
@@ -0,0 +1,48 @@
+package nl.tudelft.dnainator.ui.drawables.phylogeny;
+
+import javafx.scene.shape.Circle;
+import javafx.scene.shape.Shape;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextAlignment;
+
+/**
+ * Represents an {@link InternalNode} that has been collapsed.
+ */
+public class CollapsedNode extends AbstractNode {
+	private Text label;
+	private InternalNode wrappedNode;
+
+	/**
+	 * Create a new collapsed node.
+	 * @param wrapped the node that has been collapsed.
+	 */
+	public CollapsedNode(InternalNode wrapped) {
+		wrappedNode = wrapped;
+		wrappedNode.getChildren().forEach(child -> child.setVisible(false));
+		wrappedNode.getChildren().add(this);
+		// Bind the margin of the wrapped node to this margin, so that the layout adapts.
+		wrappedNode.marginProperty().bind(this.marginProperty());
+
+		this.label = new Text(LeafNode.LABEL_X_OFFSET, LeafNode.LABEL_Y_OFFSET, "collapsed.");
```

Comment:
OCD but I would prefer to have the dot removed, e.g.: `collapsed`. :stuck_out_tongue: